k9mail-library/src/main/java/com/fsck/k9/mail/ServerSettings.java

```diff
@@ -16,10 +16,13 @@
  * @see com.fsck.k9.mail.store.StoreConfig#getTransportUri()
  */
 public class ServerSettings {
+
+    public enum Type { IMAP, SMTP, WebDAV, POP3 }
```

Comment:
Converting this to an enum makes it obvious that I combined things that don't really belong together. It would probably be better to create two enums `StoreType` and `TransportType` (in more appropriate locations). That also makes it necessary to have (at least) two `ServerSettings` classes. `IncomingServerSettings` and `OutgoingServerSettings`.

Also, the naming is inconsistent. Maybe it would be better to make the string that is used for import/export explicit, e.g. WEBDAV("WebDAV").